### PR TITLE
Update chiro.md

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
add publication
address https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1826

### CODE REVIEW

**Comment by cthoyt** [comment]
Blocked by #1855

File: ontology/chiro.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ products:
     name: "CHEBI Integrated Role Ontology main release in OWL format"
   - id: chiro.obo
     name: "CHEBI Integrated Role Ontology additional release in OBO format"
+publications: 
+   - id: https://chemrxiv.org/engage/chemrxiv/article-details/60c74d1f9abda21f6df8d3ce
```
Comment (cthoyt) [inline ontology/chiro.md:26]: I am SUPER upset with the changes to chemRxiv, I will probably never use this preprint server again - they removed the identifiers from the webpage for viewing. Total nonsense. Instead we should solely use the DOI

```suggestion
   - id: https://doi.org/10.26434/chemrxiv.12591221
```

**Comment by cthoyt** [comment]
Spacing needs fixing

**Review by matentzn — APPROVED** [review]

**Comment by nicolevasilevsky** [comment]
thanks @matentzn and @cthoyt

**Pull request merged**